packages/pages/src/common/src/template/internal/getTemplatesFilepath.ts

```diff
@@ -0,0 +1,37 @@
+import glob from "glob";
+import path from "path";
+import { Path } from "../../project/structure.js";
```

Comment:
nit but would it make sense to move Path to its own file?

Reply:
sure